### DISCUSSION TITLE
Does xvector in mtex refer to the sample coordinate system or the acquisition surface coordinate system?

### DISCUSSION THREAD

**Discussion by xiu-shen** [mtex-toolbox/mtex#2251] [Ask Anything]
SF_array(i) = slipSystems(i).SchmidFactor(inv(ori) * xvector);

Does xvector refer to the sample coordinate system or the acquisition surface coordinate system?

**Comment by ralfHielscher** [comment]
In MTEX both reference frames are assumed to be the same, see [EBSDReferenceFrame](https://mtex-toolbox.github.io/EBSDReferenceFrame.html)

Ralf.

**Reply by xiu-shen** [reply]
I've gone through it and what I understand is that xvector refers to the space coordinate system x

**Reply by ralfHielscher** [reply]
And how do you think, one can specify a position at the acquisition surface coordinate system?
